stories/topics/proc-aap-aws-application-upgrade.adoc

```diff
@@ -0,0 +1,150 @@
+. Main variables.
+.. *aws_foundation_stack_name*: The name of the foundation deployment. This is the same name you used when you deployed the foundation.
+.. *aws_region*: The region where the foundation deployment is located.
+.. *aws_ssm_bucket_name*: The name of the S3 bucket where the configuration files for the AWS SSM are stored. Can be an existent bucket used of AWS SSM or a new one
+. The AWS Backup. The upgrade process will create a backup of your existing deployment before the main process starts. It requires the following variables:
+.. *aws_backup_vault_name*: The name of the AWS Backup vault to use. The vault must exist in the same region as the deployment.
+.. *aws_backup_iam_role_arn*: The ARN of the IAM role to use for the backup. The role must have the `AWSBackupServiceRolePolicyForBackup` policy attached to it. If you want to use the default AWS Backup service role, use the following ARN: `arn:aws:iam::<YOUR_ACCOUNT_NUMBER>:role/service-role/AWSBackupDefaultServiceRole`.
+
+The final result should look like this:
+
+[source,bash]
+----
+$ tree
+.
+├── extra_vars/
+│   └── extra_vars.yml
+└── secrets/
+    ├── config
+    └── credentials
+----
+
+== Performing the upgrade
+
+The following instructions describe how to run the upgrade playbook.
+
+Pull the Docker image for Ansible on Clouds operational container having tag with the version which you want to upgrade to.
+
+[NOTE]
+=====
+The Ansible on Clouds operational image tag must match with the version which you want to upgrade to. For example, if your foundation deployment version is 2.2.0, pull operational image with tag 2.2.20230215 to upgrade to version 2.2.20230215.
+=====
+
+[source,bash]
+----
+$ export IMAGE=<image-path>
+$ docker pull $IMAGE
+----
+
+Running the upgrade playbook as a container.
+
+Replace `<deployment-name>` with your foundation deployment name. Use the following command to run the playbook.
+
+[source,bash]
+----
+# Export the volume paths
+$ export VOLUME_AWS_CREDENTIALS_DIR=<absolute path to workdir>/secrets
+$ export VOLUME_EXTRA_VARS_DIR=<absolute path to workdir>/extra_vars
+
+$ docker run --rm \
+    --env PLATFORM=AWS \
+    --env DEPLOYMENT_NAME=<deployment-name> \
+    --env GENERATE_INVENTORY=true \
+    -v ${VOLUME_AWS_CREDENTIALS_DIR}:/home/runner/.aws/:rw \
+    -v ${VOLUME_EXTRA_VARS_DIR}:/extra_vars:ro \
+    ${IMAGE} \
+      redhat.ansible_on_clouds.aws_upgrade \
+      -e @/extra_vars/extra_vars.yml
+----
+
+After successfully running the playbook, the playbook will return something like this
+
+[source,bash]
+----
+TASK [redhat.ansible_on_clouds.standalone_aws_upgrade : [upgrade] [LOG] upgrade version] ***
+ok: [localhost] => {
+    "msg": "Successfully upgraded from '2.2.20221201-00' -> '2.2.20230215-00'."
+}
+----
+
+Your {AAPonAWS} deployment now should be upgraded to newer version and you should now be able to successfully log in to {PlatformName} {ControllerName} and {HubName} using your deployment credentials.
```

Comment:
. Your {AAPonAWS} deployment is  now upgraded to a newer version and you can log in to ...